lib/vrl/stdlib/Cargo.toml

```diff
@@ -75,6 +75,7 @@ default = [
     "assert_eq",
     "boolean",
     "ceil",
+    "chunks",
```

Comment:
I'm assuming this should be a default feature, but please let me know if this is not the case.